core/email_api.php

```diff
@@ -1549,11 +1549,13 @@ function email_format_bug_message( array $p_visible_bug_data ) {
 			$t_time_tracking = '';
 		}
 
+		$t_access_level_string = ' - ';
 		if( user_exists( $t_bugnote->reporter_id ) ) {
 			$t_access_level = access_get_project_level( $p_visible_bug_data['email_project_id'], $t_bugnote->reporter_id );
-			$t_access_level_string = ' (' . get_enum_element( 'access_levels', $t_access_level ) . ') - ';
-		} else {
-			$t_access_level_string = '';
+			# Only display access level when higher than 0 (ANYBODY)
```

Comment:
I opened new PR https://github.com/mantisbt/mantisbt/pull/778 where I introduced a string for users with no access.